### PR TITLE
Added appveyor support and the badge to the readme

### PR DESCRIPTION
**Pull request by jjvilleg**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+environment:
+  matrix:
+        - jdk: "C:\\Program Files\\Java\\jdk1.8.0\\bin:"
+
+build_script:
+  # Build the compiled extension
+  - "gradlew.bat install check -Prunit=online -DwlpVersion=17.0.0.3 --stacktrace --info"
+
+test: off
```
Comment (cthigh) [inline appveyor.yml]: What automated tests does this turn off?  Are the tests running?

File: src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy [hunk 2]
```diff
@@ -155,11 +153,8 @@ class Liberty implements Plugin<Project> {
             description 'Starts the WebSphere Liberty Profile server.'
             logging.level = LogLevel.INFO
             group 'Liberty'
-            dependsOn 'libertyCreate'
 
-            project.afterEvaluate {
-                if (dependsOnApps(server)) dependsOn 'installApps'
-            }
+            project.afterEvaluate { dependsOn installDependsOn(server, 'libertyCreate') }
```
Comment (cthigh) [inline src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy]: Feels like a regression here.  David recently made changes here and moved the `dependsOn 'libertyCreate'` outside of the `afterEvaluate`.

Also, recheck the `libertyStart` project.task definition.

Comment (jjvilleg) [reply src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy]: I manually brought in these changes from @mattbsox 's fork. I don't need them for this PR and can remove the lberty.groovy file from this PR.

File: appveyor.yml (hunk 1 above)
Comment (jjvilleg) [reply appveyor.yml]: The tests are running from our buildscript command. This is appveyor configuration where we could break apart our buildscript command and add the test part of the script here, but with no real benefit.

File: src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy [hunk 3]
```diff
@@ -60,7 +60,7 @@ class Liberty implements Plugin<Project> {
         }
 
         //Create expected server extension from liberty extension data
-        project.afterEvaluate {
+        project.afterEvaluate {
```
Comment (dshimo) [inline src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy]: random space but maybe im picky

Comment (cthigh) [reply src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy]: I thought Liberty.groovy was going to be removed from the PR.

**Pull request merged**